openstf/feature_engineering/apply_features.py

```diff
@@ -149,185 +55,50 @@ def apply_features(data, minute_list=None, h_ahead=24):
                             np.random.uniform(0.7,1.7, 200)))
 
     """
-    if minute_list is None:
-        minute_list = []
 
-    lag_functions = generate_lag_functions(data, minute_list, h_ahead)
+    # Get lag feature functions
+    lag_feature_functions = generate_lag_feature_functions(feature_set_list, horizon)
```

Comment:
@JanMaartenvanDoorn Why is it `generate_lag_feature_functions`  and `create_holiday_feature_functions`? If what there functions do is similar its probably more clear to use the same verb.